func/functions_example_problems.py

```diff
@@ -0,0 +1,102 @@
+########################## Example Problems ##########################
+
+# MAKES TWENTY: Given two integers, return True if the sum of the integers is 20 or if one of the integers is 20.
+# If not, return False
+
+def makes_twenty(num1, num2):
+    if (num1 + num2 == 20) or num1 == 20 or num2 == 20:
+        return True
+    else:
+        return False
+
+# ANIMAL CRACKERS: Write a function takes a two-word string and returns True if both words begin with same letter
+
+# animal_crackers('Levelheaded Llama') --> True
+# animal_crackers('Crazy Kangaroo') --> False
+
+def animal_crackers(string):
+    for idx, letter in enumerate(string):
+        if letter == " ":
+            if string[idx + 1] == string[0]:
+                return True
+            else:
+                return False
+
+
+# MASTER YODA: Given a sentence, return a sentence with the words reversed
+
+# master_yoda('I am home') --> 'home am I'
+# master_yoda('We are ready') --> 'ready are We'
+
+def master_yoda(string):
+   string_list = string.split(" ")
+   #string_list = ["I", "am", "home"]
+   final_string = ""
+   for index in range(len(string_list)-1, -1, -1):
+        final_string = final_string + string_list[index] + " "
+        print(final_string) 
+   print(final_string)     
+
+
+# PAPER DOLL: Given a string, return a string where for every character in the original there are three characters
+def paper_doll(string):
+    word = ""
+    for letter in string:
+        word = word + letter + letter + letter
```

Comment:
@eddyannobil So this works, but it's a bit tacky :D. Is there a way to achieve this without doing `letter + letter + letter`? Imagine we wanted to repeat each letter 1000 times, will you write 1000 'letter'?